libraries.json

```diff
@@ -221,6 +221,13 @@
     "description": "Open source framework for building truly native mobile apps with Angular, Vue.js, TypeScript, or JavaScript.",
     "language": "JavaScript"
   },
+  "Neo4j" : {
+    "imports": ["neo4j"],
```

Comment:
As I can see the imports in JavaScript are done like this:
`const neo4j = require('neo4j-driver')`
so it is supposed to be `neo4j-driver` not `neo4j`.